packages/client/hmi-client/src/components/model/petrinet/model-diagrams/tera-model-diagram.vue

```diff
@@ -190,15 +190,15 @@ async function renderGraph() {
 		graphLegendColors.value = renderer.depthColorList;
 	}
 
-	renderer.on('node-click', (_eventName, _event, selection) => {
-		const { id, data } = selection.datum();
-		if (data.type === NodeType.Transition) {
-			selectedTransitionId.value = id;
-			openValueConfig.value = true;
-		}
-	});
-
 	if (isStratified.value) {
+		renderer.on('node-click', (_eventName, _event, selection) => {
+			const { id, data } = selection.datum();
+			if (data.type === NodeType.Transition) {
+				selectedTransitionId.value = id;
+				openValueConfig.value = true;
+			}
+		});
+
```

Comment:
This seems to be fixing a different problem than the one described by the PR ??
If so, can you move it out? Just so commit is itself a cohesive unit.

Reply:
sure